app/views/browse/mobile-clients.html

```diff
@@ -14,7 +14,7 @@ <h1 class="contains-actions">
                class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
                data-toggle="dropdown"><i class="fa fa-ellipsis-v" aria-hidden="true"></i><span class="sr-only">Actions</span></a>
             <ul class="dropdown-menu dropdown-menu-right actions action-button">
-              <li ng-if="ctrl.mobileCIEnabled && (ctrl.mobileClient.spec.clientType === 'android' || ctrl.mobileClient.spec.clientType === 'iOS')">
+              <li ng-if="ctrl.mobileCIEnabled && (ctrl.mobileClient.spec.clientType === 'android' || ctrl.mobileClient.spec.clientType === 'iOS' || ctrl.mobileClient.spec.clientType === 'cordova')">
```

Comment:
I'd imagine we will get rid of this check when we have all app types covered, so no too concerned about it.